app/files.js

```diff
@@ -312,6 +331,58 @@ const roomArchived = {
   }
 }
 
+const userInvited = {
+  key: "userInvited",
+  noun: "User",
+  display: {
+    label: "User Joined",
+    description: "Triggers when a user invited to Room."
+  },
+  operation: {
+    inputFields: [
+      {
+        label: "Room",
+        key: "id",
+        required: true,
+        dynamic: "roomCreated.id.title"
+      },
+      {
+        label: "active",
+        key: "active",
+        type: "boolean",
+        helpText: "Return only those who are active"
+      }
+    ],
+    /**
+     * @param {ZObject} z
+     * @param {Bundle<SessionAuthenticationData, UserJoinedFields>} bundle
+     * @returns {Promise<User[]>}
+     */
+    async perform(z, bundle) {
+      /**
+       * @param {UsersList} data
+       * @param {boolean} activationStatus
+       * @returns {User[]}
+       */
+      const usersByList = function (data, activationStatus) {
+        return data
+          .filter(item => activationStatus ? item.sharedTo.activationStatus === 1 : true)
+          .map(item => item.sharedTo)
+      }
+
+      const client = new Client(bundle.authData.baseUrl, z.request)
+      const files = new FilesService(client)
+      const filters = {
+        filterType: 0 // Only users
```

Comment:
I would suggest putting this into a constant. By doing so, you will have a self-explanatory construction that doesn't need comments.

```js
const ONLY_USERS_FILTER_TYPE = 0
// or
const OnlyUsersFilterType = 0
```

Reply:
> I would suggest putting this into a constant. By doing so, you will have a self-explanatory construction that doesn't need comments.
> 
> ```js
> const ONLY_USERS_FILTER_TYPE = 0
> // or
> const OnlyUsersFilterType = 0
> ```

Added a ONLY_USERS_FILTER_TYPE constant